### PR TITLE
A better implementation of face expressions detection .

### PR DESCRIPTION
**Pull request by VarunChopra11**
A better implementation of face expressions detection Using a Web Streamlit UI.

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": 2,
+    "builds": [
+      {
+        "src": "app.py",
+        "use": "@vercel/python"
```
Comment (manish-9245) [inline vercel.json:6]: Is your site deployed on Vercel? If yes please drop the link
I do not see a functional link on your repo

File: app.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+        emotion = result[0]['dominant_emotion']
+
+        cv2.rectangle(frame, (x, y), (x + w, y + h), (0, 0, 255), 2)
+        cv2.putText(frame, emotion, (x, y - 10), cv2.FONT_HERSHEY_SIMPLEX, 0.9, (0, 0, 255), 2)
+
+    return frame
+
+def main():
+    st.title("Real-time Emotion Detection")
```
Comment (manish-9245) [inline app.py:27]: If you plan to make a Streamlit UI on top of my implementation, please implement separation of concern.
Try to keep the code decentralized

**Review by manish-9245 — CHANGES_REQUESTED** [review]
Address the comments

File: vercel.json (hunk 1 above)
Comment (VarunChopra11) [reply vercel.json:6]: Currently working on the deployment, will provide you the link soon .

**Comment by VarunChopra11** [comment]
closed

**Pull request closed without merging**